runtime/vm/vmprops.c

```diff
@@ -530,6 +530,107 @@ getLibSubDir(J9JavaVM *vm, const char *subDir, char **value)
 }
 
 #define COM_SUN_MANAGEMENT "com.sun.management"
+
+UDATA
+loadCMDLineSystemProperties(J9JavaVM *vm, JavaVMInitArgs *initArgs)
```

Comment:
Is this refactoring still needed?  I was kind of expecting it to be removed as we agreed to only support the Java-level ::setProperties call.

Reply:
no Ive removed the change